examples/tests/valid/json.w

```diff
@@ -146,7 +146,7 @@ let jsonElements = Json {
   }
 };
 
-if let val = jsonElements.tryGet("strings")?.tryGet("single")?.tryAsStr() {
+if let val = jsonElements.tryGet("strings")?.tryGet("single")?.asStr() {
```

Comment:
Could you please explain why you made that change?

Reply:
tryAsStr() returns `str?` while asStr() returns `str`. The change proves that the optionality of the previous calls to tryGet() carries forward into asStr(), which makes the file return value `str?`

Reply:
Got you, thanks!